bundle/deploy/terraform/import.go

```diff
@@ -0,0 +1,92 @@
+package terraform
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/log"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+type BindOptions struct {
+	AutoApprove  bool
+	ResourceType string
+	ResourceKey  string
+	ResourceId   string
+}
+
+type importResource struct {
+	opts *BindOptions
+}
+
+// Apply implements bundle.Mutator.
+func (m *importResource) Apply(ctx context.Context, b *bundle.Bundle) error {
+	dir, err := Dir(ctx, b)
+	if err != nil {
+		return err
+	}
+
+	// If the bundle.tf.json file does not exist, write it.
+	// This is necessary because the import operation requires the resource to be defined in the Terraform configuration.
+	_, err = os.Stat(filepath.Join(dir, "bundle.tf.json"))
+	if err != nil {
+		err = bundle.Apply(ctx, b, Write())
```

Comment:
Can't we always write it? Note that the Terraform interpolation mutator also needs to run.

Reply:
@pietern doing this will include all local changes to bundle config as well which I believe we should not include especially considering we push the state remotely